urwid/listbox.py

```diff
@@ -82,8 +82,28 @@ def get_prev(self, position):
             return None, None
 
 
+    def next_position(self, position):
+        """
+        Return position after start_from.
+        """
+        if len(self) - 1 <= position:
+            if self.wrap_around:
+                return 0
+            raise IndexError
+        return position + 1
+
+    def prev_position(self, position):
+        """
+        Return position before start_from.
+        """
+        if position <= 0:
+            if self.wrap_around:
+                return len(self) - 1
+            raise IndexError
+        return position - 1
+
```

Comment:
Uhhh no, I don't think this belongs here. Even despite that moving these two methods into `ListWalker` deduplicates code from the two descendant classes, `SimpleListWalker` & `SimpleFocusListWalker`.

Two issues:
 1. `ListWalker` is abstract, sort-of like an interface: we can't assume that `position` is of numeric type, and that we can do arithmetic on it. An imaginary `MoonWalker` descendant might be able to use filenames as `position`-s, for example.
 2. `self.wrap_around` is unknown in the scope of `ListWalker`. This will certainly aggravate static analysis tools (pylint, mypy, etc). This will also break other people's custom ListWalker implementations if we merge it.

I'm okay with the `wrap_around` feature — but this little refactoring needs to be undone.